client/src/components/BookingList/BookingList.tsx

```diff
@@ -0,0 +1,22 @@
+import { Typography } from '@mui/material';
+import { Booking as IBooking } from '../../interface/Booking';
+import Booking from './Booking';
+
+export interface PropTypes {
+  bookings: IBooking[];
+  type: string;
+}
+
+const BookingList = ({ bookings, type }: PropTypes) => {
+  return (
+    <>
+      <Typography sx={{ marginTop: '10px', fontVariant: 'small-caps', fontWeight: 'bold' }}>
+        {type} bookings:
+      </Typography>
+      {bookings.map((b) => (
+        <Booking key={b.start.getTime()} booking={b} />
```

Comment:
Let's make sure we use unique indexes as keys in React. If two pending bookings have the same start time we will no longer have unique keys for our component

Importance of unique indexes in React: https://betterprogramming.pub/why-react-keys-matter-an-introduction-136b7447cefc